ibis/tests/expr/test_table.py

```diff
@@ -2082,3 +2082,67 @@ def test_unbind_with_namespace():
 
     assert s.op() == expected.op()
     assert s.equals(expected)
+
+
```

Comment:
Would love to see test cases for every ifelse statement in the implementation:
- bools (raw, no ibis.literal(bool))
- ints (raw)
- lambda
- a Table

Reply:
Also a test for deferreds, eg _.bind("a")

Reply:
Table was already there, added tests for the rest.
`bind(_, "column_name")` case isn't supposed to work since `bind()` expects an actual table expression not a deferred as the first argument. Adding a test case for `bind(t, _.column_name)`.